src/main/java/org/folio/circulation/infrastructure/storage/feesandfines/FeeFineActionRepository.java

```diff
@@ -50,7 +51,9 @@ public CompletableFuture<Result<FeeFineAction>> create(StoredFeeFineAction feeFi
   }
 
   public CompletableFuture<Result<FeeFineAction>> findById(String id) {
+    log.debug("findById:: parameters id: {}", id);
     if(isNull(id)) {
+      log.warn("findById:: id is null");
       return ofAsync(() -> null);
     }
```

Comment:
Add more logging to findChargeActionForAccount(Account account), createAll(Collection<StoredFeeFineAction> feeFineActions) methods